specification/mixedreality/data-plane/Microsoft.MixedReality/preview/0.2-preview.0/mr-aoa.json

```diff
@@ -0,0 +1,451 @@
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "BlobUploadEndpoint_Get": {
+            "$ref": "examples/GetBlobUploadEndpoint.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "JobStatus": {
+      "enum": [
+        "NotStarted",
+        "Running",
+        "Succeeded",
+        "Failed",
+        "Cancelled"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "JobStatus",
+        "modelAsString": false
+      }
+    },
+    "Vector3": {
+      "required": [
+        "x",
+        "y",
+        "z"
+      ],
+      "type": "object",
+      "properties": {
+        "x": {
+          "format": "float",
+          "type": "number"
+        },
+        "y": {
+          "format": "float",
+          "type": "number"
+        },
+        "z": {
+          "format": "float",
+          "type": "number"
+        }
+      }
+    },
+    "Quaternion": {
+      "required": [
+        "x",
+        "y",
+        "z",
+        "w"
+      ],
+      "type": "object",
+      "properties": {
+        "x": {
+          "format": "float",
+          "type": "number"
+        },
+        "y": {
+          "format": "float",
+          "type": "number"
+        },
+        "z": {
+          "format": "float",
+          "type": "number"
+        },
+        "w": {
+          "format": "float",
+          "type": "number"
+        },
+        "isIdentity": {
+          "type": "boolean",
+          "readOnly": true
+        }
+      }
+    },
+    "Pose": {
+      "required": [
+        "rotation",
+        "translation"
+      ],
+      "type": "object",
+      "properties": {
+        "rotation": {
+          "$ref": "#/definitions/Quaternion"
+        },
+        "translation": {
+          "$ref": "#/definitions/Vector3"
+        }
+      }
+    },
+    "Vector4": {
+      "required": [
+        "x",
+        "y",
+        "z",
+        "w"
+      ],
+      "type": "object",
+      "properties": {
+        "x": {
+          "format": "float",
+          "type": "number"
+        },
+        "y": {
+          "format": "float",
+          "type": "number"
+        },
+        "z": {
+          "format": "float",
+          "type": "number"
+        },
+        "w": {
+          "format": "float",
+          "type": "number"
+        }
+      }
+    },
+    "IngestionConfiguration": {
+      "description": "Represents an ingestion configuration.",
+      "required": [
+        "gravity",
+        "scale"
+      ],
+      "type": "object",
+      "properties": {
+        "dimensions": {
+          "$ref": "#/definitions/Vector3",
+          "x-nullable": true
```

Comment:
`x-nullable` indicates an explicit JSON `null` on the wire (sent/received).  Does the service send/receive JSON `null`s?  Question applies to all `x-nullable: true` statements.

Reply:
Yes, our service accepts an explicit `null` for those fields, and may return an explicit `null` for those fields as well.